tests/onnxruntime/testing_utils.py

```diff
@@ -49,16 +49,14 @@
     "deberta": "hf-internal-testing/tiny-random-DebertaModel",
     "deberta-v2": "hf-internal-testing/tiny-random-DebertaV2Model",
     "deit": "hf-internal-testing/tiny-random-DeiTModel",
-    "donut": "fxmarty/tiny-doc-qa-vision-encoder-decoder",
+    "vision-encoder-decoder-donut": "fxmarty/tiny-doc-qa-vision-encoder-decoder",
     "detr": "hf-internal-testing/tiny-random-detr",
     "dinov2": "hf-internal-testing/tiny-random-Dinov2Model",
     "distilbert": "hf-internal-testing/tiny-random-DistilBertModel",
     "dpt": "hf-internal-testing/tiny-random-DPTForSemanticSegmentation",
     "electra": "hf-internal-testing/tiny-random-ElectraModel",
-    "encoder-decoder": {
-        "hf-internal-testing/tiny-random-EncoderDecoderModel-bert-bert": ["text2text-generation"],
-        "mohitsha/tiny-random-testing-bert2gpt2": ["text2text-generation", "text2text-generation-with-past"],
-    },
+    "encoder-decoder": "optimum-internal-testing/tiny-random-encoder-decoder-gpt2-bert",
+    "encoder-decoder-bert-bert": "hf-internal-testing/tiny-random-EncoderDecoderModel-bert-bert",
```

Comment:
this way the test doesn't need to loop over multiple tests in the same test scope.